ansible/group_vars/edusharing.yml

```diff
@@ -177,6 +182,32 @@ alfresco_ccmodel_add_custom_properties:
 # 
 edu_custom_licenses:
 
+
+# define config for classifications keywords
+# After activated the classification-keywords, you need to add an edu-sharing job in "edu_migration_replace_mapping" variable
+# example: 
+# edu_migration_replace_mapping:
+    # - jobClass: "org.edu_sharing.repository.server.jobs.quartz.ImportFactualTermMarc21"
+    #  parameters:
+    #    - fileUri: '{{ edu_sharing_protocol | default("http") }}://{{edu_sharing_host}}/oer/{{classification_keywords_file}}'
+
+# activate the classification-keyword (true/false)
+activate_classification_keywords: false
+# activate the classification-keyword update (true/false)
+activate_classification_keywords_update: false
+# define the classification-keyword interval in days = 1 means every day
+classification_keywords_update_interval: 
+# give the path to the file where you want download the classification-keywords
+# must be public folder: example: /var/www/oer/ 
+classification_keywords_dir: 
+# define the classification-keyword file name example: classification_keywords.xml
+classification_keywords_file: 
```

Comment:
Variables that are unlikely to be changed by the user should be given a sensible default value and should also not be set in the general configuration if possible (this unnecessarily complicates the config file). Instead, it is better to set them in the `defaults` of the role.

Reply:
@mirjan-hoffmann 
I refactor variables and use defaults, for variable that maybe user will not change them